.github/stale.yml

```diff
@@ -0,0 +1,18 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 365
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 30
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - pinned
+  - security
+# Label to use when marking an issue as stale
+staleLabel: wontfix
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because there has been no
+  activity in the past year.  It will be closed automatically if no further
+  activity occurs in the next 7 days.  Feel free to re-open at any time if this
```

Comment:
should be 30?

Reply:
I just copied on the from emscripten so I guess that wrong too ?  :)

Reply:
In that case yes, sounds like that's wrong too, heh...